EU4toV2/Source/Mappers/CultureGroupsMapper/CultureGroupsMapping.cpp

```diff
@@ -0,0 +1,27 @@
+#include "CultureGroupsMapping.h"
+#include "../../Configuration.h"
+#include "CommonRegexes.h"
+#include "ParserHelpers.h"
+
+mappers::CultureGroupsMapping::CultureGroupsMapping(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void mappers::CultureGroupsMapping::registerKeys()
+{
+	registerKeyword("v2", [this](std::istream& theStream) {
+		v2 = commonItems::getString(theStream);
+	});
+	registerKeyword("hpm", [this](std::istream& theStream) {
+		hpm = commonItems::getString(theStream);
+		if (theConfiguration.isHpmEnabled())
```

Comment:
Is the configuration coming in as a global? It'd be clearer if a boolean was passed in here instead.